data-processing-lib/src/data_processing/data_access/data_access_local.py

```diff
@@ -145,8 +148,10 @@ def _get_input_files(
                 path=input_path, cm_files=cm_files, min_file_size=min_file_size, max_file_size=max_file_size
             )
 
-        input_files = set(self._get_all_files_ext(path=input_path, extensions=self.files_to_use))
-        output_files = set(self._get_all_files_ext(path=output_path, extensions=self.files_to_use))
+        input_files = set(self._get_all_files_ext(path=input_path, extensions=self.files_to_use, absolute_paths=False))
+        output_files = set(
+            self._get_all_files_ext(path=output_path, extensions=self.files_to_use, absolute_paths=False)
+        )
         input_only_files = input_files - output_files
```

Comment:
This is completely wrong. we should always use absolute paths, as to different directories can have a file with the same name